### PR TITLE
achurchard/style fixes

### PR DESCRIPTION
**Pull request by Cyclam**
Style fixes to conform with `azdev style`. 

[Work item 810679 ](https://dev.azure.com/msazuredev/AzureForOperators/_workitems/edit/810679)

Manual testing done by running some CLI commands.

### CODE REVIEW

File: src/aosm/azext_aosm/deploy/deploy_with_arm.py [hunk 1]
```diff
@@ -594,14 +596,18 @@ def validate_and_deploy_arm_template(
         if validation_res.error:
             # Validation failed so don't even try to deploy
             logger.error(
-                "Template for resource group %s has failed validation. The message was: %s.\
-                See logs for additional details.",
+                (
+                    "Template for resource group %s has failed validation. The message"
+                    " was: %s.                See logs for additional details."
```
Comment (jamiedparsons) [inline src/aosm/azext_aosm/deploy/deploy_with_arm.py:601]: Is that the formatting that you want?

File: src/aosm/azext_aosm/deploy/deploy_with_arm.py [hunk 2]
```diff
@@ -594,14 +596,18 @@ def validate_and_deploy_arm_template(
...
+                ),
                 resource_group,
                 validation_res.error.message,
             )
             logger.debug(
-                "Template for resource group %s failed validation. \
-                Full error details: %s",
+                (
+                    "Template for resource group %s failed validation.                "
```
Comment (jamiedparsons) [inline src/aosm/azext_aosm/deploy/deploy_with_arm.py:608]: Again, why so many spaces?

File: src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py [hunk 3]
```diff
@@ -466,14 +466,16 @@ def get_chart_mapping_schema(
         )
         if not os.path.exists(mappings_path):
             raise InvalidTemplateError(
-                f"ERROR: The helm package '{helm_package.name}' does not have a valid values mappings file. \
-                    The file at '{helm_package.path_to_mappings}' does not exist.\n\
-                    Please fix this and run the command again."
+                f"ERROR: The helm package '{helm_package.name}' does not have a valid"
+                " values mappings file.                     The file at"
+                f" '{helm_package.path_to_mappings}' does not exist.\n                 "
+                "   Please fix this and run the command again."
```
Comment (jamiedparsons) [inline src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py:472]: Again are the spaces correct here?

File: src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py [hunk 4]
```diff
@@ -466,14 +466,16 @@ def get_chart_mapping_schema(
...
             )
         if not os.path.exists(values_schema):
             raise InvalidTemplateError(
-                f"ERROR: The helm package '{helm_package.name}' is missing values.schema.json.\n\
-                Please fix this and run the command again."
+                f"ERROR: The helm package '{helm_package.name}' is missing"
+                " values.schema.json.\n                Please fix this and run the"
```
Comment (jamiedparsons) [inline src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py:477]: And again

File: src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py [hunk 5]
```diff
@@ -489,14 +491,16 @@ def get_chart_mapping_schema(
             new_schema = self.search_schema(deploy_params_dict, schema_data)
         except KeyError as e:
             raise InvalidTemplateError(
-                f"ERROR: There is a problem with your schema or values for the helm package '{helm_package.name}'. \
-                    Please fix this and run the command again."
+                "ERROR: There is a problem with your schema or values for the helm"
+                f" package '{helm_package.name}'.                     Please fix this"
```
Comment (jamiedparsons) [inline src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py:495]: again

File: src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py [hunk 6]
```diff
@@ -698,8 +695,9 @@ def get_chart_name_and_version(
 
         if not os.path.exists(chart):
             raise InvalidTemplateError(
-                f"There is no Chart.yaml file in the helm package '{helm_package.name}'. \
-                    Please fix this and run the command again."
+                "There is no Chart.yaml file in the helm package"
+                f" '{helm_package.name}'.                     Please fix this and run"
```
Comment (jamiedparsons) [inline src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py:699]: And again

File: src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py [hunk 7]
```diff
@@ -709,8 +707,9 @@ def get_chart_name_and_version(
                 chart_version = data["version"]
             else:
                 raise FileOperationError(
-                    f"A name or version is missing from Chart.yaml in the helm package '{helm_package.name}'. \
-                        Please fix this and run the command again."
+                    "A name or version is missing from Chart.yaml in the helm package"
+                    f" '{helm_package.name}'.                         Please fix this"
```
Comment (jamiedparsons) [inline src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py:711]: ditto

**Review by jamiedparsons — COMMENTED** [review]
Looks good - thanks for doing this!

Could you add some notes to the README?  What should a developer be running before merging to check linting?  Are there tools to automatically fix things up?

File: src/aosm/azext_aosm/deploy/deploy_with_arm.py (hunk 2 above)
Comment (Cyclam) [reply src/aosm/azext_aosm/deploy/deploy_with_arm.py:608]: Sorry, and thanks for catching. I think that was the merge from main messing things up :-(

File: src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py (hunk 3 above)
Comment (Cyclam) [reply src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py:472]: Sorry, and thanks for catching. I think that was the merge from main messing things up :-(

File: src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py (hunk 4 above)
Comment (Cyclam) [reply src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py:477]: Sorry, and thanks for catching. I think that was the merge from main messing things up :-(

File: src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py (hunk 5 above)
Comment (Cyclam) [reply src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py:495]: Sorry, and thanks for catching. I think that was the merge from main messing things up :-(

File: src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py (hunk 6 above)
Comment (Cyclam) [reply src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py:699]: Sorry, and thanks for catching. I think that was the merge from main messing things up :-(

File: src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py (hunk 7 above)
Comment (Cyclam) [reply src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py:711]: Sorry, and thanks for catching. I think that was the merge from main messing things up :-(

**Comment by Cyclam** [comment]
`setup.md`, which is aimed at developers, already has a section on running `azdev style` and `azdev linter`. I've added a note that `black` is a useful tool for resolving these problems, but I don't have much beyond that.

**Pull request merged**